Sources/Mini/Dispatcher.swift

```diff
@@ -230,3 +231,26 @@ public final class DispatcherSubscription: Comparable, Disposable {
         return lhs.priority <= rhs.priority
     }
 }
+
+public class AtomicCounter {
```

Comment:
This is something internal to the library, shouldn't be exposed as public

Reply:
Fixed!